packages/fx-core/src/plugins/resource/apim/index.ts

```diff
@@ -118,7 +118,7 @@ async function _postProvision(ctx: PluginContext, progressBar: ProgressBar): Pro
     const aadConfig = new AadPluginConfig(ctx.configOfOtherPlugins);
 
     const apimManager = await Factory.buildApimManager(ctx, solutionConfig);
-    const aadManager = await Factory.buildAadManager(ctx, );
+    const aadManager = await Factory.buildAadManager(ctx,);
```

Comment:
`buildAadManager(ctx,)` => `buildAadManager(ctx)`

Reply:
got it.